pg/seed.pgsql

```diff
@@ -21,19 +21,79 @@ SET default_tablespace = '';
 SET default_table_access_method = heap;
 
 --
--- Name: tasks; Type: TABLE; Schema: public; Owner: postgres
+-- Name: journal; Type: TABLE; Schema: public; Owner: tpl2_2021h1
+--
+
```

Comment:
Adding this line here resolved my issue where it couldn't find that role

`CREATE USER tpl2_2021h1 SUPERUSER;
`